test/integration/scheduler/volume_binding_test.go

```diff
@@ -416,9 +428,26 @@ func setupCluster(t *testing.T, nsName string, numberOfNodes int) *testConfig {
 			clientset.CoreV1().PersistentVolumes().DeleteCollection(nil, metav1.ListOptions{})
 			clientset.StorageV1().StorageClasses().DeleteCollection(nil, metav1.ListOptions{})
 			cleanupTest(t, context)
-			utilfeature.DefaultFeatureGate.Set("VolumeScheduling=false,LocalPersistentVolumes=false")
 		},
 	}
+
+	if len(featureStr) != 0 {
+		config.teardown = func() {
+			clientset.CoreV1().Pods(ns).DeleteCollection(nil, metav1.ListOptions{})
+			clientset.CoreV1().PersistentVolumeClaims(ns).DeleteCollection(nil, metav1.ListOptions{})
+			clientset.CoreV1().PersistentVolumes().DeleteCollection(nil, metav1.ListOptions{})
+			clientset.StorageV1().StorageClasses().DeleteCollection(nil, metav1.ListOptions{})
+			cleanupTest(t, context)
+			utilfeature.DefaultFeatureGate.Set(featureStr)
```

Comment:
This doesn't actually disable the feature gates, it just resets them to true.  Maybe k8s.io/apiserver/pkg/util/feature/testing.SetFeatureGateDuringTest would work better here.

Reply:
Oh, really sorry for missing notification of this PR. Is this already covered by #66148 now? Or we can continue it.

Reply:
@cofyc can you work out with @resouer the work that needs to be done?

Reply:
@resouer not covered, #66148 was opened for new invalidations added in #66106, #65616 
I'll add tests for them after this is merged.

Reply:
Thanks for let me know, I will rebase and update this test soon.